elasticsearch-extensions/lib/elasticsearch/extensions/test/cluster.rb

```diff
@@ -299,23 +296,25 @@ def start
           def stop
             begin
               nodes = __get_nodes
-            rescue Exception => e
-              __log "[!] Exception raised when stopping the cluster: #{e.inspect}".ansi(:red)
+            rescue StandardError => e
+              __log "[!] Error raised when stopping the cluster: #{e.inspect}".ansi(:red)
               nil
             end
 
-            return false if nodes.nil? or nodes.empty?
+            return false if nodes.nil? || nodes.empty?
 
-            pids  = nodes['nodes'].map { |id, info| info['process']['id'] }
+            pids = nodes['nodes'].map { |id, info| info['process']['id'] }
 
-            unless pids.empty?
-              __log "Stopping Elasticsearch nodes... ".ansi(:faint), :print
+            if pids.empty?
+              return false
+            else
+              __log 'Stopping Elasticsearch nodes... '.ansi(:faint), :print
 
               pids.each_with_index do |pid, i|
-                ['INT','KILL'].each do |signal|
+                ['INT', 'KILL'].each do |signal|
                   begin
                     Process.kill signal, pid
-                  rescue Exception => e
+                  rescue StandardError => e
```

Comment:
This change is the same kind of behavior-changing style change. Revert it to the previous state.